packages/manager/apps/pci-private-network/src/pages/listing/ListingLayout.component.tsx

```diff
@@ -0,0 +1,112 @@
+import { useCallback, useEffect } from 'react';
+import { Outlet, useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import {
+  PciGuidesHeader,
+  useProjectUrl,
+  Title,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import {
+  OsdsBreadcrumb,
+  OsdsSpinner,
+  OsdsTabBar,
+  OsdsTabBarItem,
+  OsdsTabs,
+} from '@ovhcloud/ods-components/react';
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+import {
+  PciAnnouncementBanner,
+  PciDiscoveryBanner,
+  useProject,
+} from '@ovh-ux/manager-pci-common';
+import { usePrivateNetworks } from '@/data/hooks/networks/useNetworks';
+import { PrivateNetworkTabName } from './ListingLayout.constant';
+import { useActiveTab } from '@/hooks/useActiveTab/useActiveTab';
+
+const ListingLayout: React.FC = () => {
+  const { t } = useTranslation('listing');
+  const projectUrl = useProjectUrl('public-cloud');
+  const { data: project } = useProject();
+  const projectId = project.project_id;
+  const navigate = useNavigate();
+  const tab = useActiveTab();
+
+  const { isPending, data: networks } = usePrivateNetworks(projectId);
+
+  const { clearNotifications } = useNotifications();
+
+  const handleTabChange = useCallback(({ detail }: CustomEvent) => {
+    clearNotifications();
+
+    if (detail?.panel === PrivateNetworkTabName.LOCAL_ZONE_TAB_NAME) {
+      navigate('./localZone');
+    } else {
+      navigate('');
+    }
+  }, []);
+
+  useEffect(() => {
+    if (networks?.length === 0) {
+      navigate(`/pci/projects/${projectId}/private-networks/onboarding`);
+    }
+  }, [networks]);
+
+  return (
+    <>
+      <OsdsBreadcrumb
+        items={[
+          {
+            href: projectUrl,
+            label: project.description,
+          },
+          {
+            label: t('pci_projects_project_network_private'),
+          },
+        ]}
+      />
+      <div className="header my-8">
+        <div className="flex items-center justify-between">
+          <Title>{t('pci_projects_project_network_private')}</Title>
+          <PciGuidesHeader category="instances" />
+        </div>
+      </div>
+
+      <PciAnnouncementBanner projectId={projectId} />
+
+      <div className="mb-5">
+        <PciDiscoveryBanner project={project} />
+      </div>
+
+      {isPending ? (
+        <OsdsSpinner
+          inline
+          size={ODS_SPINNER_SIZE.lg}
+          className="block text-center"
+        />
+      ) : (
+        <OsdsTabs panel={tab} onOdsTabsChanged={handleTabChange}>
```

Comment:
why is there a panel here and an active property on panel side ?

Reply:
it allow to target the tab directly from url otherwise it will always be redirect to region tab